e2e/lerna-changed/lerna-changed.spec.ts

```diff
@@ -0,0 +1,316 @@
+
+        expect(output.combinedOutput).toMatchInlineSnapshot(`
+          /tmp/lerna-e2e/lerna-changed-test/modules/package-a
+          /tmp/lerna-e2e/lerna-changed-test/packages/package-c
+          lerna notice cli v999.9.9-e2e.0
+          lerna info Looking for changed packages since 0.0.0
+          lerna success found 2 packages ready to publish
+
+        `);
+      });
+    });
+
+    describe("-p", () => {
+      it("should list package-a and package-c as changed with parseable output instead of columnified view", async () => {
+        const output = await runCLI("changed -p");
+
+        expect(output.combinedOutput).toMatchInlineSnapshot(`
+          /tmp/lerna-e2e/lerna-changed-test/modules/package-a
+          /tmp/lerna-e2e/lerna-changed-test/packages/package-c
+          lerna notice cli v999.9.9-e2e.0
+          lerna info Looking for changed packages since 0.0.0
+          lerna success found 2 packages ready to publish
+
+        `);
+      });
+    });
+
+    describe("-pla", () => {
+      it("should list package-a, package-b, and package-c as changed, with version and package info, in a parseable output", async () => {
+        const output = await runCLI("changed -pla");
+
+        expect(output.combinedOutput).toMatchInlineSnapshot(`
+          /tmp/lerna-e2e/lerna-changed-test/modules/package-a:package-a:0.0.0
+          /tmp/lerna-e2e/lerna-changed-test/packages/package-b:package-b:0.0.0:PRIVATE
+          /tmp/lerna-e2e/lerna-changed-test/packages/package-c:package-c:0.0.0-alpha.1
+          lerna notice cli v999.9.9-e2e.0
+          lerna info Looking for changed packages since 0.0.0
+          lerna success found 3 packages ready to publish
+
+        `);
+      });
+    });
+
+    describe("--toposort", () => {
+      it("should list package-a and package-c as changed, but in topological order", async () => {
+        const output = await runCLI("changed --toposort");
+
+        expect(output.combinedOutput).toMatchInlineSnapshot(`
+          package-c
+          package-a
+          lerna notice cli v999.9.9-e2e.0
+          lerna info Looking for changed packages since 0.0.0
+          lerna success found 2 packages ready to publish
+
+        `);
+      });
+    });
+
+    describe("--graph", () => {
+      it("should list package-a and package-c as changed with their dependencies in a json list", async () => {
+        const output = await runCLI("changed --graph");
+
+        expect(output.combinedOutput).toMatchInlineSnapshot(`
+          {
+            "package-a": [
+              "package-c",
+              "package-d"
+            ],
+            "package-c": [
+              "package-d"
+            ]
+          }
+          lerna notice cli v999.9.9-e2e.0
+          lerna info Looking for changed packages since 0.0.0
+          lerna success found 2 packages ready to publish
+
+        `);
+      });
+    });
+  });
+
+  describe("--include-merged-tags", () => {
```

Comment:
Please include a link to the issue/PR you described here

Reply:
Added in a comment above the --include-merged-tags section